spec/features/budgets/access_show.rb

```diff
@@ -0,0 +1,13 @@
+require "rails_helper"
+
+feature "views show page" do
+  scenario "successfully" do
+    user = create(:user)
+    log_in_as user
+    budget = create(:budget)
```

Comment:
Again, I would change how the objects are create as to be more defined, less assuming

Reply:
Done sha: 1502d2e